Mathlib/Data/Int/GCD.lean

```diff
@@ -306,13 +306,13 @@ theorem gcd_mul_right (i j k : ℤ) : gcd (i * j) (k * j) = gcd i k * natAbs j :
   apply Nat.gcd_mul_right
 #align int.gcd_mul_right Int.gcd_mul_right
 
-theorem gcd_pos_of_non_zero_left {i : ℤ} (j : ℤ) (i_non_zero : i ≠ 0) : 0 < gcd i j :=
-  Nat.gcd_pos_of_pos_left (natAbs j) (natAbs_pos.2 i_non_zero)
-#align int.gcd_pos_of_non_zero_left Int.gcd_pos_of_non_zero_left
+theorem gcd_pos_of_ne_zero_left {i : ℤ} (j : ℤ) (hi : i ≠ 0) : 0 < gcd i j :=
+  Nat.gcd_pos_of_pos_left _ $ natAbs_pos.2 hi
```

Comment:
Any reason for not using the mathport output?
```suggestion
  Nat.gcd_pos_of_pos_left _ <| natAbs_pos_of_ne_zero hi
```
(and below)

Reply:
`natAbs_pos_of_ne_zero` is deprecated.

Reply:
Huh, ok then. I think sticking to `<|` is probably still better, but not a big deal.